packages/governance/test/unitTests/test-paramGovernance.js

```diff
@@ -133,7 +133,7 @@ const setUpVoterAndVote = async (committeeCreator, zoe, qHandle, choice) => {
   const invitations = await E(committeeCreator).getVoterInvitations();
 
   const seat = E(zoe).offer(invitations[0]);
-  const voteFacet = E(seat).getOfferResult();
+  const voteFacet = E.get(E(seat).getOfferResult()).voter;
```

Comment:
ditto on suggestion above to use `voter`

Reply:
done